packages/design-system/src/styles/components/_Table.scss

```diff
@@ -80,8 +95,9 @@
       // stylelint-enable selector-no-qualifying-type
       flex: none;
       font-weight: bold;
-      max-width: 120px;
-      width: 50%;
+      max-width: $table-stacked-title-max-width;
+      width: 80%;
```

Comment:
Why 80% here?

Reply:
To keep some spacing before the vertical line of the data title

Reply:
I dont quite understand, do you have a screenshot of what that means? If there's a mix-width already, wouldnt we just want the width to be 100%? or even just remove it? I dont see any difference when i remove the width: 80%

Reply:
![Screen Shot 2020-09-30 at 1 58 34 PM](https://user-images.githubusercontent.com/20322880/94723206-639ff500-0326-11eb-9776-70e13c947cbb.png)

Reply:
What about using `calc`? something like this:
`calc(100% - #{$spacer-2})`

And then also include a comment explaining why we need this here